runtime/lua/vim/lsp.lua

```diff
@@ -901,7 +901,7 @@ end
 --- Field `cmd` in {config} is required.
 ---
 ---@param config (table) Configuration for the server:
---- - cmd: (table|string|fun(dispatchers: table):table) command string or
+--- - cmd: (string[]|fun(dispatchers: table):table) command string or
```

Comment:
```suggestion
--- - cmd: (string|string[]|fun(dispatchers: table):table) command string or
```

Reply:
I thought the intention was to remove the `string` (and it also changes the `table` to `string[]`)